templates/sync-function-validation-module.js

```diff
@@ -1026,7 +1026,7 @@ function validationModule() {
       var currentItemEntry = itemStack[itemStack.length - 1];
 
       // Copy all but the last/top element so that the item's parent is at the top of the stack for the custom validation function
-      var customValidationItemStack = itemStack.slice(-1);
+      var customValidationItemStack = itemStack.slice(0, -1);
```

Comment:
This was the mistake. According to MDN's documentation for the [`slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) method, passing a negative number as the sole parameter collects everything starting at that offset from the end of the array. Since it's `-1`, that means that it always returns an array with only the last element.